services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -291,19 +354,10 @@ def engine(self) -> Engine:
         log.info("Updating project %s for user %s", project_uuid, user_id)
 
         async with self.engine.acquire() as conn:
-            joint_table = user_to_projects.join(projects)
-            query = select([projects.c.id, projects.c.uuid]).\
-                select_from(joint_table).\
-                    where(and_(projects.c.uuid == project_uuid, user_to_projects.c.user_id == user_id))
-            result = await conn.execute(query)
-
-            # ensure we have found one
-            row = await result.first()
-            if not row:
-                raise ProjectNotFoundError(project_uuid)
+            row = await self._get_study(user_id, project_uuid, exclude_foreign=['tags'])
 
```

Comment:
ok, so with regard to my previous comment, you probably also need to use _get_study from get_user_project.
Also, we should use either project or study but both is confusing especially in the same module.

Reply:
I am always using study now, but I didn't want to start refactoring naming since we are probably changing it again soon... and don't use any of those but only use "node"

Reply:
_get_study is now used from get_user_project

Reply:
and file refactored to only contain ``project``